src/main/java/org/usfirst/frc/team1758/robot/commands/CommandBase.java

```diff
@@ -1,16 +1,19 @@
 package org.usfirst.frc.team1758.robot.commands;
 
 import org.usfirst.frc.team1758.robot.subsystems.DriveTrain;
+import org.usfirst.frc.team1758.robot.subsystems.Sensors;
 import org.usfirst.frc.team1758.robot.subsystems.Vision;
 
 import edu.wpi.first.wpilibj.command.Command;
 
 public abstract class CommandBase extends Command {
```

Comment:
In this class we can add a method called `getSensors()` which returns the sensors subsystem. From there we can call `getGyroAngle()` and use that to update the SmartDashboard from `Robot.java`